<ui version="4.0" >
 <class>TestWin</class>
 <widget class="QDialog" name="TestWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Test window for OpenChrono</string>
  </property>
  <widget class="QWidget" name="verticalLayout" >
   <property name="geometry" >
    <rect>
     <x>25</x>
     <y>30</y>
     <width>41</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>-1</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>RPM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="SliderRPM" >
      <property name="maximum" >
       <number>20000</number>
      </property>
      <property name="singleStep" >
       <number>10</number>
      </property>
      <property name="pageStep" >
       <number>100</number>
      </property>
      <property name="value" >
       <number>9999</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayout_3" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>30</y>
     <width>31</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>-1</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>T2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="SliderTemp2" >
      <property name="maximum" >
       <number>200</number>
      </property>
      <property name="singleStep" >
       <number>1</number>
      </property>
      <property name="pageStep" >
       <number>10</number>
      </property>
      <property name="value" >
       <number>49</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayout_2" >
   <property name="geometry" >
    <rect>
     <x>180</x>
     <y>30</y>
     <width>31</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>-1</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>T1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="SliderTemp1" >
      <property name="maximum" >
       <number>200</number>
      </property>
      <property name="singleStep" >
       <number>1</number>
      </property>
      <property name="pageStep" >
       <number>10</number>
      </property>
      <property name="value" >
       <number>49</number>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>270</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum" >
    <number>20000</number>
   </property>
   <property name="value" >
    <number>0</number>
   </property>
   <property name="alignment" >
    <set>Qt::AlignHCenter</set>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="format" >
    <string>%v RPM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StartStop" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>30</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Start / stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="butTest" >
   <property name="geometry" >
    <rect>
     <x>80</x>
     <y>180</y>
     <width>83</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Test</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>360</x>
     <y>260</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayout_4" >
   <property name="geometry" >
    <rect>
     <x>310</x>
     <y>30</y>
     <width>265</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>0</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="1" >
       <widget class="QPushButton" name="butDown" >
        <property name="text" >
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QPushButton" name="butLeft" >
        <property name="text" >
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="butRight" >
        <property name="text" >
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="butUp" >
        <property name="text" >
         <string>Up</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="butOk" >
        <property name="text" >
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>63</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="butCancel" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblT2" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblT1" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRPM" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>210</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TestWin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TestWin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderRPM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblRPM</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>35</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel" >
     <x>53</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderTemp1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblT1</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>193</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel" >
     <x>173</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderTemp2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lblT2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>262</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel" >
     <x>271</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderRPM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>33</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel" >
     <x>93</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
